runtimes/common/Cargo.toml

```diff
@@ -30,6 +30,9 @@ sp-std = {git = "https://github.com/paritytech/substrate", default-features = fa
 default = ["std"]
 fast-gov = []
 runtime-benchmarks = [
+  "attestation/runtime-benchmarks",
+  "frame-support/runtime-benchmarks",
```

Comment:
Why does runtime common enable these features upon runtime benchmarks? I'm talking specifically about `attestation` and `parachain-staking`.